modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/query/ElasticsearchQueryTranslatorFixture.java

```diff
@@ -14,63 +14,121 @@
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_geoShapeQueryTranslator",
+			new GeoShapeQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_idsQueryTranslator",
+			new IdsQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_matchAllQueryTranslator",
+			new MatchAllQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_matchPhrasePrefixQueryTranslator",
+			new MatchPhrasePrefixQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_matchPhraseQueryTranslator",
+			new MatchPhraseQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_matchQueryTranslator",
+			new MatchQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_moreLikeThisQueryTranslator",
+			new MoreLikeThisQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_multiMatchQueryTranslator",
+			new MultiMatchQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_nestedQueryTranslator",
+			new NestedQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_percolateQueryTranslator",
+			new PercolateQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_prefixQueryTranslator",
+			new PrefixQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_rangeTermQueryTranslator",
+			new RangeTermQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_regexQueryTranslator",
+			new RegexQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_scriptQueryTranslator",
+			new ScriptQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_simpleQueryStringQueryTranslator",
+			new SimpleStringQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_stringQueryTranslator",
+			new StringQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_termQueryTranslator",
+			new TermQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_termsQueryTranslator",
+			new TermsQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_termsSetQueryTranslator",
+			new TermsSetQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_wildcardQueryTranslator",
+			new WildcardQueryTranslatorImpl());
+		ReflectionTestUtil.setFieldValue(
+			elasticsearchQueryTranslator, "_wrapperQueryTranslator",
+			new WrapperQueryTranslatorImpl());
+
+		_elasticsearchQueryTranslator = elasticsearchQueryTranslator;
```

Comment:
@W2jx Another example, please try to avoid this.

Reply:
I will fix it.